ovn_k8s/watcher/namespace_watcher.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2016 Nicira, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from ovs import vlog
+
+import ovn_k8s
+from ovn_k8s.common import kubernetes
+from ovn_k8s.common import util
+from ovn_k8s.common import variables
+import ovn_k8s.processor
+import ovn_k8s.processor.policy_processor as pp
+
+
+log = vlog.Vlog('namespace_watcher')
+
+
+class NamespaceWatcher(object):
+
+    def __init__(self, ns_stream):
+        self._ns_stream = ns_stream
+        self.ns_cache = {}
+
+    def _send_event(self, ns_name, event_type):
+        event = ovn_k8s.processor.NSEvent(event_type,
+                                          source=ns_name,
+                                          metadata=self.ns_cache[ns_name])
```

Comment:
A comment here that we are adding this to a policy processor's event queue will make it very explicit that namespace events are only processed by policy processor.